src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CollectionDataContract.cs

```diff
@@ -139,35 +140,30 @@ internal sealed class CollectionDataContract : DataContract
     {
         private XmlDictionaryString _collectionItemName;
 
-        private XmlDictionaryString _childElementNamespace;
+        private XmlDictionaryString? _childElementNamespace;
 
-        private DataContract _itemContract;
+        private DataContract? _itemContract;
 
         private CollectionDataContractCriticalHelper _helper;
 
-        public CollectionDataContract(CollectionKind kind) : base(new CollectionDataContractCriticalHelper(kind))
```

Comment:
isn't this a public type when USE_REFEMIT is defined?

Reply:
USE_REFEMIT is never defined. Will be removed with https://github.com/dotnet/runtime/issues/41512.